core/src/main/java/com/oath/oak/NovaManager.java

```diff
@@ -7,10 +7,10 @@
 
 class NovaManager implements MemoryManager {
     static final int RELEASE_LIST_LIMIT = 1024;
-    private ThreadIndexCalculator threadIndexCalculator;
-    private List<List<Slice>> releaseLists;
-    private AtomicInteger globalNovaNumber;
-    private OakBlockMemoryAllocator allocator;
+    private final ThreadIndexCalculator threadIndexCalculator;
```

Comment:
I wonder why it would be requested to make those fields final...?

Reply:
Actually, this is not part of the checkstyle, but rather an Intellij suggestion.
It makes sense to make these members final because they never change.
Making this explicit helps code readability.